core/src/main/java/tc/oc/pgm/commands/MapCommands.java

```diff
@@ -192,6 +196,16 @@ public void map(Audience audience, CommandSender sender, @Text MapInfo map) {
     }
 
     audience.sendMessage(createTagsComponent(map.getTags()));
+
+    if (PGM.get().getMapOrder() instanceof MapPoolManager) {
+      List<MapPool> mapPools = ((MapPoolManager) PGM.get().getMapOrder()).contains(map);
```

Comment:
```
String pools = ((MapPoolManager) ..).getPools().stream()
    .filter(p -> p.getMaps().contains(map)).collect(Collectors.joining(", "));
```

Reply:
.joining gives an error

Reply:
With the filter it will give me all `MapPool`s i want right. But how do i make a Collector that collects all those `MapPool`s names.

Reply:
Nvm, i think i figured it out!